src/Microsoft.PowerShell.Commands.Utility/Microsoft.PowerShell.Commands.Utility.csproj

```diff
@@ -67,6 +67,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="2.10.0" />
+    <PackageReference Include="System.Threading.AccessControl" Version="4.5.0" />
```

Comment:
This didn't add a file to the MSI?

Reply:
No, I just moved the package reference from `SDK` to `Uitlity`.